mpeg-encoder.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function () {
+  // eslint-disable-next-line
+  importScripts('https://cdnjs.cloudflare.com/ajax/libs/lamejs/1.2.0/lame.min.js')
```

Comment:
Can we use it from npm?

Reply:
no without change the current way to create a new worker (pass the function string). you can't use require without bundle it first.